reference/commands/download.rst

```diff
@@ -30,7 +30,39 @@ conan download
                             -vvv or -vtrace
       --only-recipe         Download only the recipe/s, not the binary packages.
       -p PACKAGE_QUERY, --package-query PACKAGE_QUERY
-                            Only upload packages matching a specific query. e.g:
+                            Only download packages matching a specific query. e.g:
                             os=Windows AND (arch=x86 OR compiler=gcc)
       -r REMOTE, --remote REMOTE
                             Download from this specific remote
+
+
+Downloads recipe and binaries to the local cache from the specified remote.
```

Comment:
I think adding a note here that "download" is not install, it will not retrieve transitive dependencies, not generate build-system generated files, etc. To make clear the intent of the command.